fpdf/fpdf.py

```diff
@@ -3719,6 +3720,15 @@ def preload_image(self, name, dims=None):
             info = ImageInfo(get_img_info(name, img, self.image_filter, dims))
             info["i"] = len(self.images) + 1
             info["usages"] = 1
+            info["iccp_i"] = None
+            if "iccp" in info and info["iccp"]:
+                if info["iccp"] in self.iccps:
```

Comment:
Also, could you add a call like this here please:
```
LOGGER.debug("ICC profile found for image %s - It will be inserted in the PDF document", name)
```